roles/openshift_node/tasks/upgrade/config_changes.yml

```diff
@@ -27,6 +27,18 @@
 - name: Update systemd units
   import_tasks: ../systemd_units.yml
 
+# Get stat of /etc/crio/crio.conf
+- stat: path=/etc/crio/crio.conf
+  register: crio_conf
+# Update the cri-o pause image to use latest version after an upgrade
+# if cri-o is the container runtime i.e /etc/crio/crio.conf exists
+- name: Update cri-o pause image
+  lineinfile:
+    dest: /etc/crio/crio.conf
+    line: pause_image = "{{ openshift_crio_pause_image }}"
```

Comment:
IMO `openshift_crio_pause_image` should be dedicated for role `container_runtime`. It would be hard to maintain and error-prone if we introduce the roles dependence in an invisible manner.

Probably it's better to import a cri-o configuration change file from role `container_runtime`.

Reply:
We don't want to change the whole cri-o configuration file when an upgrade happens as we don't want to overwrite settings customized by the user. The only thing we need to change is the pause image so that it reflects the version it has been upgraded to.